tests/view_tests/tests/test_static.py

```diff
@@ -124,6 +126,20 @@ def tearDown(self):
         super().tearDown()
         urls.urlpatterns = self._old_views_urlpatterns
 
+    def test_normal_result(self):
+        self.assertEqual(str(static('test')[0]), str(url('^test(?P<path>.*)$', serve)))
```

Comment:
I think checking attributes of the `LocaleRegexURLResolver` would be more straightforward than testing indirectly via the `__str__()` method.